tests/cases/fourslash/scriptLexicalStructureBindingPatterns.ts

```diff
@@ -0,0 +1,4 @@
+/// <reference path='fourslash.ts'/> 
+////var foo, {}
+////var bar, []
+verify.getScriptLexicalStructureListCount(3); // global (1) + variable declarations (2)

```

Comment:
If the binding patterns have names inside, do we show those as nav bar items? Can you add a test for that?